source/code/plugins/in_oms_syslog.rb

```diff
@@ -0,0 +1,313 @@
+#
```

Comment:
Does this come from existing in_syslog.rb fluentd code? If so is it identical, or what is new/changed?

Reply:
not identical, I've added unix domain support.
At some point we need to create a branch because we can't keep patching changes like that.